wiki/Installation/pt-br.md

```diff
@@ -0,0 +1,77 @@
+# Instalação
+
+*Veja também: [Instalação no macOS](/wiki/Installation/macOS)*
+
+Esta página irá te dizer como ter o osu! funcionando em seu dispositivo Windows. A [osu!academy](/wiki/osu!academy) tem um [vídeo tutorial](https://www.youtube.com/watch?v=0V5GwzmMhpU) que explica como instalar osu! no Windows.
+
+## Requisitos mínimos do sistema
+
+- .NET framework 4+ (o instalador deve pedir para fazer o download, se não, pode [encontrá-lo aqui](https://www.microsoft.com/en-us/download/details.aspx?id=48130))
+- Uma placa de vídeo decente.
+
+## Instalando osu!
+
+1. Vá para [página de download](https://osu.ppy.sh/home/download) e baixe o instalador.
+2. Localize e inicie o instalador.
+3. Enquanto o instalador está rodando, existe um tempo de dez segundos que permite você selecionar o local de instalação.
+   - Se quiser mudar, clique no texto embaixo da palavra `Hi!` e selecione o local de instalação.
```

Comment:
```suggestion
   - Se quiser mudar, clique no texto embaixo da palavra `Olá!` e selecione o local de instalação.
```
The installer actually changes to PT-BR if it detects the locale after downloading it, it is translated there as well. Just checked.